packages/msw-addon/src/mswDecorator.js

```diff
@@ -9,15 +9,15 @@ export function initializeWorker(options) {
   return initialize(options)
 }
 
-export function initialize(options) {
+export function initialize(options, handlers = []) {
```

Comment:
Just a small comment.. but I'm wondering if this would be better suited as an object instead of multiple parameters. Makes it easier for autocompletion when we have types, as well as avoids situations where people don't want to set options, but only handlers and they have to do like:
```ts
initialize(undefined, handlers)
```
@kettanaito WDYT?

Reply:
yep happy to do this. agree thats better 🙌 (was trying not to introduce a breaking change initially) Thanks for the feedback/review.

Reply:
You're totally right! I think for the sake of this change it might be better to not use an object and not introduce breaking changes. We can think of doing that later when we introduce typescript, then we can make a major version with some breaking changes.

Reply:
Would you mind bringing the signature back to `initialize(options, handlers)`? you had a great point and I think if we plan on bringing breaking changes it's better to do it with other contributions. Thanks!

Reply:
No problem will revert.

Reply:
All reverted 🙏